ios/ContextSDKBridge.swift

```diff
@@ -56,9 +56,9 @@ class ContextSDKBridge: NSObject {
     }
 
     @objc(calibrate:maxDelay:customSignals:withResolver:withRejecter:)
-    func calibrate(flowName: String, maxDelay: Int, customSignals: NSDictionary, resolve: @escaping RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
+    func calibrate(flowName: String, maxDelay: NSNumber, customSignals: NSDictionary, resolve: @escaping RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
```

Comment:
Hm maybe nevermind my previous comment 😅 so for RN we force users to pass a `maxDelay`, whereas for Swift we don't?